app/util/conf.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, config_yml):
         self.custom_dataset_query = self.get_property('custom_dataset_query') or ""
         self.verbose = self.settings['verbose']
         self.total_actions_per_hour = self.get_property('total_actions_per_hour')
+        self.chromedriver_version = self.obj['modules']['selenium']['chromedriver']['version']
```

Comment:
Why `self.chromedriver_version` is not in BaseAppSettings ?

Reply:
Its necessary to avoid failure on any crowd run.

Reply:
Initialize with default None?